README.md

```diff
@@ -1,5 +1,7 @@
 # darts-api
 
+Change for demo.
```

Comment:
Why do we need this?

Reply:
We don't, it was a test to show the dev env in a demo